### ISSUE TITLE
Proposal: Enabling 'Club Finder' capability

### ISSUE THREAD

**Issue by howaskew** [openactive/modelling-opportunity-data#313]
## Proposers

Howard Askew (@howaskew), Darren Temple, Tim Corby and Andrew Newman (the OpenActive team at the Open Data Institute), in relation to conversations with London Sport, Yorkshire Sport Foundation and bodies such as the Football Association (FA) and the National Small-bore Rifle Association (NSRA).

## Use Case

**As an activity provider, facility provider or sporting body,** I would like to share my data at the location or ‘club’ level. This would enable club finder style capabilities in 3rd party applications - allowing people to discover activity providers in a specific location. 

My organisation may not yet be ready to publish event or session level data or to take open bookings, but would like to take first steps toward open data.

On an activity finder, a potential participant would see a location with activity type, website links and contact details. For example a football club might list an adult men's team, junior teams at different age brackets and women's and girl's teams. Participants would then be able to get in touch to find out more and to make a booking.

**Additional Use Cases**

**As a local authority or active partnership,** I would like to better map the provision of sport and physical activity in my jurisdiction in order to:
- inform strategic decision making about the allocation of funding, resources and interventions
- more easily support national campaigns and initiatives at a local or regional level
- make comparisons with health, economic and demographic data to generate insights

**As a social prescriber or link worker,** I would like to understand the local provision of clubs accredited to national governing bodies in order to be better assured that the opportunities I refer my clients to have accountability related to safeguarding, risk management, coaching qualifications etc.

## Why is this not covered by existing properties?

The existing properties could cover this proposed use case. For example, the [FacilityUse](https://developer.openactive.io/data-model/types/facilityuse) and/or [IndividualFacilityUse](https://developer.openactive.io/data-model/types/individualfacilityuse) data model types could represent Football Pitches at a club site. However, the current guidance for officially accepted feeds currently [suggests requirement of additional timed event level data](https://developer.openactive.io/publishing-data/data-feeds/types-of-feed#slot-based-events-facilityuses) i.e. the [Slot](https://developer.openactive.io/data-model/types/slot) data model type.

## Please provide a link to example data

[Examples 4 and 5](https://openactive.io/modelling-opportunity-data/EditorsDraft/#example-4-adding-a-venue-and-an-organizer) in the specification link organisations, locations and activities.

The [data visualiser](https://visualiser.openactive.io/) 'map' tab shows examples of existing data which can be presented in a ‘club finder’ style.

## Related initiatives

London Sport are currently (October 2023) running a project to collect community club data across London. This includes:
 - surveying organisations to capture the minimum / required fields to create an ‘open club standard’. They are exploring how this can align with existing standards including OpenActive.
- Developing tools for central / bulk capture of club data, that may be useful beyond the initial focus area of London.

There are many opportunities for collaboration here to maximise impacts of this work and the OA team’s work as we have also been exploring options to upload in bulk from spreadsheet / csv to an RPDE style feed.

## Proposals

Once club data is collected, how can it be disseminated to maximise impact? If it can be shared as open data alongside existing OpenActive feeds, it can be available for display in 3rd party tools like activity finders and directories.

Here are some initial options to achieve this, we welcome comments and alternative approaches:

**Option 1**
A new feed type for clubs, requiring activity type, location, organisation and contact details, but not requiring event level data. 

Not recommended, as:
We do not wish to add another level of  complexity to the feed specifications and documentation, and
By utilising an existing feed type it could simplify the process of adding event level data at a later date.

**Option 2**
Use the existing SessionSeries or FacilityUse feed types to provide information on the activities or facilities at a location or provided by an organisation. This will require revised guidance on handling Series and Facilities without specific Sessions or Slots subevent data.

Clarifying the documentation around feeds is already on the ‘to-do’ list, so this may seem like a promising approach, however we’d need to explore the impacts on data consumers.

**Option 3**
Use the existing SessionSeries or FacilityUse feed types with a dummy subevent for each activity or facility. The event will have an end date in the future.

Again this will require updated guidance, however it may prove easier to implement/manage from a data consumer perspective.

**Option 4**
Use the existing SessionSeries or FacilityUse feed types, without subevents, but with the addition of a new boolean field like "isClub".

**Option 5**
Instead of RPDE style feeds, use a potentially simpler format for dissemination for slowly changing data - something like [CSV on the web](https://csvw.org/guides/why-use-csvw.html). This embeds metadata into a csv file to provide information on how the data should be interpreted.

## Example

[The NSRA currently lists some information about their clubs in tabular format](https://www.nsra.co.uk/index.php/home/clubs/clubfinder). (The map works intermittently). They would like some of this information to be visible alongside OpenActive opportunity data in 3rd party activity finders.

## Broader Issues

A. In a situation where individual clubs can upload their information to a central portal, and NGBs are starting to provide bulk uploads of information for affiliated clubs, how do we tackle data quality issues like accuracy, duplication and currency?

B. With no event level data, there is no direct opportunity to receive revenue from bookings, so there is potentially less financial incentive for some organisations in the data ecosystem to handle this additional data transfer. However, we feel there is potential value to be found elsewhere in providing an easier entry to the world of open data, to support upskilling and increased maturity in data and digital in the sector, and to simplify the move to event level data and booking revenue as the sector matures and the market grows.

**Comment by nickevansuk** [comment]
Referencing previous proposal #295 on the same subject

**Comment by citizenfish** [comment]
Very interesting analysis there Nick I'd add a few things to your proposal, which makes huge sense to me.

a) many clubs consider their Facebook page to be their website, some pedantry probably required to define "website"

b) JSON-LD is way beyond the vast majority of website mangers, most of whom are frantically fighting Wordpress with limited technical skills. My view is this needs to be completely obfuscated from them via a simple toolset that aids the creation and testing of it.

c) I'll probably phrase this badly but I wondered if the club hasn't got a website but has got a club finder entry, then in effect they have got a website as there is an online electronic record pointing to their details, my poorly formed idea is that in the absence of a url a placeholder would format their club finder record as a simple HTML page.

d) I see where you are coming from with the phrase "encouraging clubs to have decent websites" but we need to recognise that in the main these are not created or maintained by IT experts. They are often cobbled together by enthusiasts frantically searching the internet for the easiest and best looking option.  This got me wondering whether their is value in an Openactive variant of Google Search Console which tests your website for "fitness for Openactive publishing". I'd love to build this.

e) my personal view is that everything possible should be done to get the least technically aware clubs listed as these are the missing entries in the online world and currently the social networks/village noticeboards own this data. 

In summary I like your proposal for publishing a catalogue of URLS, as it simplifies discovery for me. The barrier to consuming this data is low and the work to get a standard in place should be relatively straightforward and hence quick.

**Comment by howaskew** [comment]
Thanks for the contributions so far on this. Starting with @nickevansuk's points...

**Previous efforts:** The data visualiser tool gives us a convenient view on to the quality of feeds and we'll start to follow up with England Squash and British Orienteering as resource allows. Table Tennis 365 doesn't have a live dataset site, and hasn't for some time, so I'm proposing to remove it from the catalogue.

**Ambitions:** I think the ambition has evolved a little beyond "get open data published and data users will come". We know there are data users. This is about helping organisations progress along the journey towards digital transformation.

**Primary vs secondary data:** To clarify in this case:
Primary club data would be data provided and maintained directly by an individual club.
Secondary club data would be lists of clubs provided by NGBs, local authorities, other sources.

**Value and Trust:** This is related to the quality and currency of the data, rather than the primary or secondary status. I agree that in general primary / operational data should be better maintained. But it could be that some NGBs provide well maintained, accurate lists of clubs. Can we base the inclusion on quality (including maintenance) rather than source?

More generally in terms of trust in OpenActive data, I think we are already in a position where there is a wide range of quality and accuracy and currency of data in the OA feeds. Effectively, the London Sport work could act as a pilot where we can explore any impacts from including club data. 

**Data Publishing Approach:** I think a consideration here is not just accuracy, but usability. Presenting club data in a format that can be read in alongside session data seems sensible. @Reikyo has prototyped a simple tool to transform a table / spreadsheet list into a 'compliant' RPDE feed. This would mean combining the club data with session data in existing tools is trivial. (Data users - please correct me if I am oversimplifying this.) 

**Modelling:** Fully agree on the semantics being key, and on the principle of differentiation.

**Digital Transformation:** Agree it is critical to learn from the past, but initiatives like London Sport's current project are happening and we need to make the most of such opportunities, share our learning and try to manage challenges and risks as best we can. We are contributing to wider data and digital strategies in Sport where we can.

**NIck's Proposal which we can call 'Option 6':** Clubs add standardised information as json-ld in their websites, and the website urls are listed in a json catalog somewhere. Nick makes many valid points about improving SEO and nudging towards digital transformation. However, it seems that this approach falls one step short of actually presenting club data in a format that allows it to be presented in existing RPDE driven OpenActive tools.

@citizenfish's enhancements - a tool to obfuscate the json-ld creation, a tool to create a holding page from json-ld content, a tool to validate websites in terms of OpenActive - are great but they don't tackle that. 

**So, Option 7:** Alongside this approach, OpenActive could host a version of Darren's tool to supply the extracted json-ld club data as an RPDE feed, that users can choose to incorporate with existing tools with little or no extra effort. This could bring new club data into, for example, Active Partnerships existing activity finder tools.

**Comment by Julz-YSF** [comment]
_> As Nick suggested, seeing the actual data fields being considered would be helpful in this discussion. So, here is the latest draft from London Sport.
> 
> Club Name.._

Would suggest adding 
>AKA - lots of clubs will have sport acronyms ARLFC some will spell it out 
>campaign/Keyword -  free back end fields than can include campaign tags or things like local authority areas etc
>siteID - Sport England's Active places ref for mapping locations, it covers maybe 90% of venues   
>text - so clubs can add an intro or "sessions most Thursdays"
>admin email - the public contact is not always the main admin contact

**Comment by howaskew** [comment]
Logging a comment from Tom Marley at W3C session earlier in the year:
Should a 'club' be an 'organiser type'? 
Others might be 'coach'

**Comment by howaskew** [comment]
[The topic was discussed 19th February 2025](https://youtu.be/R1nIfpacnhk?si=y6zba1kjjXpJWY6-&t=2178)

Steven Taylor mentioned an item that is not currently included but could be relevant here:
Last Updated Date

Andrew Newman mentioned an approach that is not explicitly listed above:
OpenActive could host and manage a 'register' of clubs, in the way that they maintain vocabularies like the activity list.

**Comment by Julz-YSF** [comment]
YSF have been working with Nautoguide to scrape public data for our directory of South Yorks and West Yorks clubs

We had 3,500 in our directory (covering a pop of 3.5m), easily doubling that.

https://www.dropbox.com/scl/fi/qw5mm9u5df7px7cx66vri/Published-data-sources.png?rlkey=cbu6nwyg7rsrzxvgz0victpo9&st=69t8pas5&dl=0 

Where possible we want to use NGB data, but align with sportsuite as that links with user crm and allows admin level access to upload Openactive feed.

Hoping to use Sport Englands, Active Places "Site ID" tag as the key to strip duplicates.  That way any number of organisations could host a register made up of multiple open data sources - like we are trying to do.

**Comment by howaskew** [comment]
A few references to related work. Sport Ireland created a national database of affiliated clubs. (They recognised many clubs were not in the position to post dated, timed events). Data available as an ESRI feature layer for use in mapping tools. 

Fields:
OBJECTID 
GlobalID 
Name (
EircodePostcode
Latitude 
Longitude 
OnMapType 
Activity 
Description 
ClubAffiliation 
Under18Membership 
NumberOfMembers 
Address 
Website 
DisabilityCharter (Yes/No)
AccessibilityDetails 
Notes 
Category 
County 
Masters 
RecordType (Club)
DataOwner 
sourceglobalid 

Data on clubs & groups is provided by the  National Governing Bodies of Sport  (NGBs). 

- If an accredited club is missing or has incorrect information, please contact the relevant NGB and request them to improve the data.
- If a non-accredited club is missing, the club should contact the NGB and register with them in order to be included.
- If a sport/activity does not have an official NGB, we regret to say that the club/group cannot be included at this time. In future we may expand the provision of data, so if you have an umbrella body that may provide and maintain information, please let us know and we can consider this in time…

Viewing the map, there are similar issues with missing data e.g. empty 'accessibilityDetails'

**Comment by howaskew** [comment]
From [Open Athletics data model](https://w3c.github.io/opentrack-cg/spec/model/#sports-club):

<p style="margin: 1em 0px; color: rgb(0, 0, 0); font-family: sans-serif; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">A<span> </span><dfn id="dfn-sports-club" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn" class="respec-offending-element" title="Found definition for &quot;Sports Club&quot;, but nothing links to it. This is usually a spec bug!" style="cursor: pointer; font-weight: bolder; text-decoration: underline wavy red;">Sports Club</dfn>,<span> </span><strong>sport club</strong><span> </span>or<span> </span><strong>athletics club</strong>, is an<span> </span><a href="https://w3c.github.io/opentrack-cg/spec/model/#organization" style="color: rgb(3, 69, 117); text-decoration-color: rgb(112, 112, 112); text-decoration-skip-ink: none;">Organization</a><span> </span>for<span> </span><a href="https://w3c.github.io/opentrack-cg/spec/model/#athlete" style="color: rgb(3, 69, 117); text-decoration-color: rgb(112, 112, 112); text-decoration-skip-ink: none;">Athletes</a><span> </span>formed for the purpose of playing sports. Clubs may have different<span> </span><a href="https://w3c.github.io/opentrack-cg/spec/model/#sports-team" style="color: rgb(3, 69, 117); text-decoration-color: rgb(112, 112, 112); text-decoration-skip-ink: none;">Teams</a><span> </span>created for specific competitions, such as leagues, relay races, etc.</p><ul style="margin-left: 0px; padding-left: 2em; color: rgb(0, 0, 0); font-family: sans-serif; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li style="margin: 0.25em 0px 0.5em; padding: 0px;">Class:<span> </span><code style="font-family: Menlo, Consolas, &quot;DejaVu Sans Mono&quot;, Monaco, monospace; font-size: 0.9em; hyphens: none; text-transform: none; text-align: start; font-variant: normal; orphans: 3; widows: 3; break-before: avoid;">Club</code><span> </span>(<code style="font-family: Menlo, Consolas, &quot;DejaVu Sans Mono&quot;, Monaco, monospace; font-size: 0.9em; hyphens: none; text-transform: none; text-align: start; font-variant: normal; orphans: 3; widows: 3; break-before: avoid;">schema:SportsClub</code>)</li><li style="margin: 0.25em 0px 0.5em; padding: 0px;">subClassOf:<span> </span><code style="font-family: Menlo, Consolas, &quot;DejaVu Sans Mono&quot;, Monaco, monospace; font-size: 0.9em; hyphens: none; text-transform: none; text-align: start; font-variant: normal; orphans: 3; widows: 3; break-before: avoid;">schema:Organization</code></li></ul>

Key | Property | Description | Value Type
-- | -- | -- | --
identifier | schema:identifier | Unique character string to identify a club. | Text
name | schema:name | Descriptive name of a club. | Text
alternateName | schema:alternateName | An alias to name a club. | Text
address | schema:address | Main postal address where a club is registered or located. | Postal Address or Text
image | schema:image | Picture of a club. | URL
logo | schema:logo | Logo or flag of the club. | URL
email | schema:email | Main email address of the club. | Text
telephone | schema:telephone | Telephone number of a club | Text
faxNumber | schema:faxNumber | Fax number of a club | Text
url | schema:url | Club's homepage. | URL
dissolutionDate | schema:dissolutionDate | Date when a team was dissolved. | Date
foundingDate | schema:foundingDate | Date when a team was established. | Date
organization | schema:memberOf | Organization which a club is attached to. | Organization
sponsor | schema:sponsor | Sponsor of a club. | Person or Organization
athlete | schema:athlete | Athlete who is affiliated to a club. | Athlete
team | schema:member | Sports Team attached to a club. | Sports Team

<div class="example" id="example-18" style="padding: 0.5em; border-left: 0.5em solid rgb(224, 203, 82); border-bottom: 0.25em solid rgb(224, 203, 82); break-inside: avoid; margin: 1em 0px; border-top-color: rgb(224, 203, 82); border-right-color: rgb(224, 203, 82); background: none 0% 0% / auto repeat scroll padding-box border-box rgb(252, 250, 238); color: rgb(0, 0, 0); counter-increment: example 1; overflow: auto; clear: both; position: relative; font-family: sans-serif; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="marker" style="text-transform: uppercase; padding-right: 0em; display: block; color: rgb(87, 75, 15);"><a class="self-link" href="https://w3c.github.io/opentrack-cg/spec/model/#example-18" style="color: rgb(3, 69, 117); text-decoration-color: rgb(112, 112, 112); text-decoration-skip-ink: none;">Example<bdi><span> </span>18</bdi></a></div><pre aria-busy="false" style="font-family: Menlo, Consolas, &quot;DejaVu Sans Mono&quot;, Monaco, monospace; font-size: 0.9em; hyphens: none; text-transform: none; text-align: start; font-variant: normal; orphans: 3; widows: 3; break-before: avoid; margin-top: 1em; margin-bottom: 1em; overflow: auto;"><code class="hljs json" style="font-family: Menlo, Consolas, &quot;DejaVu Sans Mono&quot;, Monaco, monospace; font-size: 14.4px; hyphens: none; text-transform: none; text-align: start; font-variant: normal; orphans: 3; widows: 3; break-before: avoid; --base: #fafafa; --mono-1: #383a42; --mono-2: #686b77; --mono-3: #717277; --hue-1: #0b76c5; --hue-2: #336ae3; --hue-3: #a626a4; --hue-4: #42803c; --hue-5: #ca4706; --hue-5-2: #c91243; --hue-6: #986801; --hue-6-2: #9a6a01; display: block; overflow-x: auto; padding: 0.5em; color: rgb(56, 58, 66); background: none 0% 0% / auto repeat scroll padding-box border-box rgb(250, 250, 250);"><span class="hljs-punctuation">{</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@context"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://w3c.github.io/opentrack-cg/contexts/opentrack.jsonld"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@id"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://w3c.github.io/opentrack-cg/examples/club/NEB.jsonld"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@type"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"Club"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"name"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"Newham &amp; Essex Beagles"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"alternateName"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"BeaglesAC"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"url"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://www.newhamandessexbeagles.co.uk/"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"image"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://example.org/image.png"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"logo"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://example.org/logo.png"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"telephone"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"(+44) 020 7511 6463"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"faxNumber"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"(+44) 020 7511 4477"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"email"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"fakeemail@example.org"</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"address"</span><span class="hljs-punctuation">:</span> <span class="hljs-punctuation">{</span>
        <span class="hljs-attr" style="color: rgb(152, 104, 1);">"name"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"Terence McMillan Stadium - Newham Leisure Centre"</span><span class="hljs-punctuation">,</span>
        <span class="hljs-attr" style="color: rgb(152, 104, 1);">"streetAddress"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"281 Prince Regent Lane"</span><span class="hljs-punctuation">,</span>
        <span class="hljs-attr" style="color: rgb(152, 104, 1);">"addressLocality"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"London"</span><span class="hljs-punctuation">,</span>
        <span class="hljs-attr" style="color: rgb(152, 104, 1);">"postalCode"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"E13 8SD"</span><span class="hljs-punctuation">,</span>
        <span class="hljs-attr" style="color: rgb(152, 104, 1);">"addressCountry"</span><span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"countrycode:GBR"</span>
    <span class="hljs-punctuation">}</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"sponsor"</span><span class="hljs-punctuation">:</span> <span class="hljs-punctuation">{</span> <span class="hljs-attr" style="color: rgb(152, 104, 1);">"name"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"Asics"</span> <span class="hljs-punctuation">}</span> <span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"athlete"</span><span class="hljs-punctuation">:</span> <span class="hljs-punctuation">[</span>
        <span class="hljs-punctuation">{</span> <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@id"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://w3c.github.io/opentrack-cg/examples/athlete/001Farah.jsonld"</span> <span class="hljs-punctuation">}</span><span class="hljs-punctuation">,</span>
        <span class="hljs-punctuation">{</span> <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@id"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://w3c.github.io/opentrack-cg/examples/athlete/002Athlete2.jsonld"</span> <span class="hljs-punctuation">}</span><span class="hljs-punctuation">,</span>
        <span class="hljs-punctuation">{</span> <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@id"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://w3c.github.io/opentrack-cg/examples/athlete/003Athlete3.jsonld"</span> <span class="hljs-punctuation">}</span>
    <span class="hljs-punctuation">]</span><span class="hljs-punctuation">,</span>
    <span class="hljs-attr" style="color: rgb(152, 104, 1);">"memberOf"</span> <span class="hljs-punctuation">:</span> <span class="hljs-punctuation">{</span> <span class="hljs-attr" style="color: rgb(152, 104, 1);">"@id"</span> <span class="hljs-punctuation">:</span> <span class="hljs-string" style="color: rgb(66, 128, 60);">"http://w3c.github.io/opentrack-cg/examples/federation/GBR.jsonld"</span> <span class="hljs-punctuation">}</span>
<span class="hljs-punctuation">}</span></code></pre></div>

**Comment by citizenfish** [comment]
Fax number in, social links not ;-) think the model may need a review..

**Comment by howaskew** [comment]
I think we have broad agreement on some principles here - which I will aim to formalise at the W3C Community Group calls in May and June. These are:

- There is a demand for information about sports clubs that is accurate and up to date. Improving consistency in the presentation or publication of this data will make it easier to consume and develop tools and services to connect people with clubs and activities..
- Club data represents another type of opportunity for sport and physical activity, so alignment with the OpenActive Opportunity data model should be taken into consideration. The model already includes 'organiser' and data feeds already reference many clubs.
- We recognise that many clubs do not have the data and digital skills or booking systems to enable publication of dated and timed events, individual sessions or facility slots. 

Given that: 

- OpenActive as an initiative aims to improve access to high-quality data about sport and physical activity opportunities, by creating a set of open data standards for use across the sector.
- There is an impetus to shift beyond the initial, foundational 'find and book' use case, to promote connections to health, wellbeing, social prescribing and to promote accessibility, inclusion and diversity.

**I **propose** we more formally recognise and update the guidance to reflect 3 different levels of opportunity data:**

1. Organisation/club providing activities
2. Organisation/club providing activities at individual sessions or slots
3. Organisation/club providing activities at bookable individual sessions or slots 

While level 3 might be the ultimate goal, offering the most seamless user experience by removing barriers to finding suitable activities and getting more active - we have to recognise the data and digital maturity in the sector. Level 1 offers a stepping stone into open data publishing, a low barrier to entry, allowing us to grow awareness of the initiative and to promote the benefits of levels 2 and 3.

If the community agree on that approach, then there are a few approaches we can take to present guidance and recommendations for implementations.

- A new, short and simple OpenActive club specification, with (optional) guidance for embedding JSON-LD into web page. 
This spec would equally support manual / offline collection of club data in bulk from National Governing Bodies, London Sport's implementation of club data management in their Open Sessions tool, and spreadsheet or .csv approaches such as the one piloted by Darren.
- An update to the [OpenActive Opportunity Data](https://openactive.io/modelling-opportunity-data/) specification, expanding the representation of organisation/club in the data model, with (optional) guidance for representing club data via RPDE feeds. This would support ongoing publication of up-to-date information about affiliated clubs from National Governing Bodies, paving the way for the addition of sessions and slots as the clubs data skills and systems evolve.
 
I think both approaches are valid and actually complement each other. For the latter, we would ideally iterate a solution with a working group including some pilot NGBs and existing data publishers.

**Comment by Julz-YSF** [comment]
I am right at the limits of my understanding here, but I feel Sport England's Active Places' 'Site ID' would be worth considering in any standard.

1. It probably locates over 90% of England venues
2. Its actively managed with annual checks 
3. Sport England uses data for things line moving communities https://www.sportengland.org/research-and-data/data/moving-communities
4. If multiple organisations like NGBs, volunteering agencies, local authorities all publish opendata *about the same club*, the **siteid** could be the first call to deduplicate: 

- Oakwell FC, 
- Oakwell Football and Community Club, 
- Oakwell AFC, 
- Oakwell Amateur and junior football club

**Comment by howaskew** [comment]
As per email to W3C mailing list just now:

We are pleased to announce a revised [Editors Draft (version 2.1)](https://openactive.io/modelling-opportunity-data/EditorsDraft/) of the OpenActive Modelling Opportunity Data specification.


The editors draft includes non-breaking changes that will have no impact on existing data feeds and applications. The full implementation of OpenActive enables a seamless end-to-end discovery and booking of opportunities - and this is very much still the prime focus of the specification.


Summary of changes:


1) Enabling organiser (or club) level data feeds


Over the last few years, we have discussed the lack of a standard means to share information about organisations that provide opportunities for sport and physical activity.


Over the last 12 months, we have explored how the OpenActive opportunity data specification can support this. For more background, see the discussion under the issue [‘Proposal: Enabling Club Finder capability’](https://github.com/openactive/modelling-opportunity-data/issues/313).


The changes to reflect organiser level feeds are non-breaking changes. They mainly consist of edits to reflect the idea that a club providing activities can be considered an 'opportunity' even if the details of specific dated and timed events are not published, or not yet published. Some introductory text in the abstract explains this. 


2) Earlier work in the Editors Draft to incorporate the Route Guide specification has been amended to point to the [standalone specification](https://openactive.io/route-guide/EditorsDraft/) and to indicate for example how a route guide could be attached to a 'guided walk' event. This has been done to reduce the size and complexity of the opportunity specification and to allow the Route Guide specification to evolve at its own pace.


3) The data model image has been updated to reflect events linked to route guides and organisers linked to activities.


4) To avoid confusion and simplify editing, in the specification we use Organizers and Organization throughout (rather than mixing in Organisers and Organisations). This aligns with schema.org, the basis of the OpenActive specification.


As per our usual process, we will now pilot these changes and seek feedback from the community before convening a W3C Community Group session to approve an update to the published Final Community Group Report.

---
Some additional tasks ahead:

- Update the standalone 'club spec' to become a guidance document with JSON-LD and RPDE examples
- Update references and text across the main website and developer sites 
- Review namespaces, including the rt namespace for route guides

**Comment by Julz-YSF** [comment]
If* I am understanding this correctly...

1. Can we publish and import club directories in an open format?
2. Is this part of Moving Communites work? 
3. Is there scope to use Active places SiteID to help deduplicate? 

-----
*<font size="huge">

**Comment by orsoladm** [comment]
> If* I am understanding this correctly...
> 
> 1. Can we publish and import club directories in an open format?
> 2. Is this part of Moving Communites work?
> 3. Is there scope to use Active places SiteID to help deduplicate?
> 
> *

Hi. To respond to your questions:
1) Yes, people will be able to publish an OpenActive feed with clubs data and anyone will be able to access the feeds that are published 
2) It's not part of MC work but we are working with 4Global to standardise the common data fields in OA and MC so people will be able to quickly and easily link and overlay datasets from the two initiatives [this is just the first step towards data interoperability across data initiatives in the sector]
3) Yes, we are working on this as well. If you have already done that work - or attempted to - it would be great to know your experience and what use case that would support.

**Comment by Julz-YSF** [comment]
Superb! 

My comment about siteID comes from the difference between data:

**Activities** will (mostly) be published by a single source/provider 
**Club databases** will be published in multiple places by governing bodies, local authorities, volunteering groups etc 

We are sport england active partnership and use Sportsuite to hold [activity finder](https://yorkshire.sportsuite.co.uk/activities/activities) and [club database](https://yorkshire.sportsuite.co.uk/directory?query=&sport=&radius=50&postcode=S72+7LP&district=&date=-1&from=&to=&skill=&age_range%5Bmin_age%5D=&age_range%5Bmax_age%5D=&disability=&campaign=&programme=).

The activity finder pulls in openactive, but the organisation database is hosted by ourselves - importing new feeds would be great, but there would be duplication if:
RFL publishes "East Leeds RL"
LCC calls it "East Leeds Rugby League Club"
VAL has it as "East Leeds Rugby League Association"

As Sport England uses SiteID in Active Places data, it seemed a ready made URN to help filter.

We used Nautoguide to scrape data following a discussion with ODI, we would probably need to only publish our held data if there wasn't a NGB feed that we pulled in.

**Comment by orsoladm** [comment]
That's correct, ideally everyone would describe data properties the same way.

For **activity name**, we encourage people to use the [OpenActive activity list](https://activity-list.openactive.io/en.html) - I am working with 4Global to align theirs to ours so everyone in the industry uses the same

For **location**, we are looking at options to add Active Places ID to the OpenActive feeds automatically to remove the burden for publishers to add one more property. In the meantime, we might provide some guidance for developers on how to do it. 

For **Club/organization** name, we've thought about the best way to implement publishing that balances accuracy of data, sustainability of maintenance of the data and speed of publishing. To speed up publishing, someone like an AP or NGB would create the dataset for clubs, as they probably already have that list, but then give the ability to individual clubs to edit the information for their organisation, as primary data entry is potentially more accurate. To make that possible, there would ideally be an authoritative way to describe the name. Are you aware of anything like that?

**Comment by Julz-YSF** [comment]
**Activity** An accepted activity list is a solid resource, although in practice I would usually include a keyword search as people are messy.

I have seen finders that will let you scroll a long list to get to zumba - and then you have to choose 11 types of [Zumba](https://activity-list.openactive.io/en/alphabetical_concepts/z.html), and you still won't see Aqua Zumba as its on the other end of the alphabet. :D

**Location**: Good to include Active Places data - I think its the most robust and actively managed dataset we have.

 **Club/organisation name** Sportsuite has an AKA field which allows a manual way of spelling out initialisisms

Look forward to seeing this go out and how Sportsuite can share our database in our counties.
